ansible/slave.yml

```diff
@@ -34,8 +34,8 @@
     - name: ensure the build dir exists
       file: path=/home/{{ jenkins_user }}/build state=directory owner={{ jenkins_user }}
 
-    - name: ensure the build dir has the right owner permissions
-      file: path=/home/{{ jenkins_user }}/build state=directory owner={{ jenkins_user }}
+    - name: ensure the home dir has the right owner permissions
+      file: path=/home/{{ jenkins_user }} state=directory owner={{ jenkins_user }} group={{ jenkins_user }} recursive=yes
```

Comment:
We do admin@trusty-huge--9b6e6c57-9d1f-411e-bb44-fd5c00cb7865:~$ grep jenkins /etc/group
jenkins-build:x:1001: